src/commands.ts

```diff
@@ -525,3 +525,31 @@ export const openCmd = async (scriptId: any) => {
     process.exit(0);
   }
 };
+
+/**
+ * Acts as a router to apis subcommands
+ * Calls functions for list, enable, or disable
+ * Otherwise returns an error of command not supported
+ */
+export const apis = async () => {
+  const list = async () => {
+    await checkIfOnline();
+    const {data} = await discovery.apis.list({
+      preferred: true,
+    });
+    data.items.forEach((api) => {
```

Comment:
Never write `forEach`, just use `for`:
```ts
for (let api of data.items) {
```

There are articles like these:
https://coderwall.com/p/kvzbpa/don-t-use-array-foreach-use-for-instead

Reply:
Oh I had no idea! Thanks for the information. There's a couple other places where we find `forEach()` (in `clasp status`) that should be changed in a separate PR.